### ISSUE TITLE
Suggestion: Inconclusive Moderator Review action.

### ISSUE THREAD

**Issue by CJH87** [bethylamine/soupcan#64]
Where a moderator has spent time reviewing an account and would not have another moderator waste their time while there were unreviewed entries in the queue.

For accounts where the the user of the account doesn't appear to like trans or LGBT people very much but that after between 5 to 10 or more minutes of analysis the only transphobia is through inferrence on their replies when read with context to the tweet chain those replies reside in.

Accounts [not exclusively for accounts which meet this following definition] where there is **SPORADIC** transphobia in RT and/or their likes, but their replies and original tweets/QT do not explicitely cause harm - might be trolling, but not transphobia when *applied inferrence is absent*.

(1) I would propose a button "Inconclusive" under actions with the tooltip along the lines of "Inconclusive after 5-10 minutes of account analysis by moderator - 'I can't appeal, but won't approve' ". 

(2) This button would flag the report with a highlight colour, lets say very faded out yellow or orange, indicative that a moderator reviewed the report plus user account and concluded they could neither appeal nor approve. 

(2a) In effect saying to other moderators "re-review this account ONLY after the report queue is up to date".

(3) The name of the moderator would be shown in the "Who reported" list. "{mod name} Inconclusive review. [-25%] {timestamp}"

(3a) The named moderator's trust 100 would NOT be attached to the report.

(4) a negative trust modifier would be applied to the report at -25%. 

(4a) Allowing community user reports to remain relevant but placing a marginally higher burden. 

(4b) Existing reporters would not lose trust.

Don't forget this is for accounts that the moderator couldn't appeal but **WON'T leave in the queue to WASTE other moderator time**.

(5) The only potential issue I see is if another moderator wants to disagree and the sum of their 100% trust and any other reporters doesn't reach 125%.

(5a) Perhaps the report *should* add weight to the first moderator's indecision and if another moderator reports the user the report remains yellow until, the report reaches 125%.

The main purpose of this proposal is to preserve the value of moderator review time. If a moderator spends 5-10 minutes on an account but cannot appeal, delete or validate. Their time still has value.

Now, one might ask, is this just a crutch for an inexperienced moderator? Could be.

I'm confident in the appeals I've issued but I've also returned about 6 reports to the mod queue with no action because they were on their way to violation but just not there yet. 

That act of not being able to decide hurts the whole process. I'm CONFIDENT that if appealed those accounts would get back onto the mod queue in the future, but I also do NOT see enough to flag them and I feel the reporting user is correctly asserting but isn't justified. 

The TOS makes it clear that, like SE, being mean (whilst non-dehumanisng) to/about/of trans people isn't enough.

**Comment by CJH87** [comment]
Seems I forgot to write suggestion in the title. ;)

**Comment by CJH87** [comment]
Talked this over with another user, I believe they correctly assert that if a second moderator also came to the same conclusion, that they also couldn't easily flag the account red nor feel comfortable appealing the account should be flagged red to play on the safe side. 

The primary goal being preserving moderator time above all else.

**Comment by CJH87** [comment]
This may not be needed with the moderation guidelines.

**Comment by bethylamine** [comment]
Closing in favor of #126
